smurfs/src/components/Smurf.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import Form from './Form';
```

Comment:
You are not using the Form component here, you should remove this import statement.